house_screen.py

```diff
@@ -76,10 +75,9 @@ def update(self):
         self.thirst_bar.bar_drain()
         self.hunger_bar.bar_drain()
         self.happiness_bar.bar_drain()
-        if self.health_bar.hp == 0 or self.thirst_bar.hp == 0:
-            self.health_bar.bar_drain()
-        elif self.health_bar.hp == 0 and self.thirst_bar.hp == 0:
+        if self.hunger_bar.hp == 0 and self.thirst_bar.hp == 0:
             self.health_bar.bar_drain()
```

Comment:
@Donnyd88 , I was looking at this code, due to the logical contradiction I removed one if statement